docs/guides/cloud/integrations/slack.mdx

```diff
@@ -1,228 +1,143 @@
 ---
-title: Integration for Slack
+title: Cypress App for Slack
 sidebar_position: 40
 sidebar_label: Slack
 ---
 
-The Cypress integration for Slack allows you to see your Cypress test results
-directly in your teams' Slack channels.
+One of our most-requested Cypress Cloud features, the Cypress app for Slack provides real-time results for your Cypress tests, all in one place—improving remote collaboration and giving wider visibility into test behavior and Application Quality.
 
-<DocsImage
-  src="/img/guides/cloud/integrations/slack/cypress-slack-integration-channel-feed.jpg"
-  alt="Cypress notification feed in Slack channel"
-  width={500}
-/>
+The Cypress app for Slack allows you to:
 
-## Install the Cypress integration for Slack
+- Improve cross-team collaboration by instantly surfacing Cypress results to the teams that need visibility
+- Confirm that key tests pass prior to launching new products or features
+- Reduce the time it takes to catch failed tests
 
-:::caution
+<DocsVideo
+  src="https://www.youtube.com/embed/hnDPz7dognY?si=PRRy4f5n6f1jzfVJ"
+  title="Cypress app for Slack"
+/>
 
-<strong>Ownership Requirements</strong>
+## Getting started
 
-In order to install the Cypress integration for Slack, you must be an admin or
-owner of both your Cypress Cloud organization and your Slack workspace.
+You need just three things to get started:
 
-:::
+- Cypress Cloud account
+- Ability to configure Cypress Cloud integrations - you must be an admin or the owner of your Cypress Cloud organization.
+- Ability to authorize Slack apps - by default, Slack allows any workspace member to authorize apps, however workspace owners may require app approvals before installation. See this [Slack help article](https://slack.com/help/articles/202035138-Add-apps-to-your-Slack-workspace) for more details.
 
-**To install the Cypress integration for Slack:**
+## Install the Cypress app for Slack
 
-1. Go to Cypress Cloud
-   [Organizations page](https://cloud.cypress.io/organizations) or open the
-   organization switcher.
-2. Select the organization you wish to integrate with Slack.
+1. In the Cypress Cloud, visit the Integrations page for the organization you wish to integrate with Slack.
+2. Click the **Enable** button in the Slack section.
+3. A popup window will be displayed requesting permission for Cypress to access the
+   workspace.
+4. After you approve the app permissions, you can choose your Slack workspace and channel to
+   associate with the installation.
+5. The installation is complete!
 
-   <DocsImage
-     src="/img/guides/cloud/organizations/select-cypress-organization.jpg"
-     alt="Select an organization"
-     width={500}
-   />
+Now Cypress will post run results for all projects in your organization to the specified Slack channel.
```

Comment:
Can you do the setup without it turning on all results for the entire org to start? Like can you enable it per project to start? Or is this step required to turn it on for an org entirely.